app/RangeTest/README.md

```diff
@@ -0,0 +1,14 @@
+# RadioTest
+
+#### set up the mote
```

Comment:
can you confirm that this is on the transmitter mote, i.e. NOT the one you attach this app to?